src/core/server/saved_objects/mappings/lib/get_types.ts

```diff
@@ -17,12 +17,14 @@
  * under the License.
  */
 
+import { EsMappings } from './types';
+
 /**
- *  Get the names of the types defined in the EsMappingsDsl
+ *  Get the names of the types defined in the EsMappings
  *
- *  @param  {EsMappingsDsl} mappings
- *  @return {Array<string>}
+ *  @param  {EsMappings} mappings
+ *  @return {string[]}
  */
-export function getTypes(mappings) {
+export function getTypes(mappings: EsMappings): string[] {
```

Comment:
Added a type here.

Reply:
Same here, I think we should avoid over documenting this function by declaring it's return type in the function signature.